base/Dependencies.toml

```diff
@@ -278,7 +278,7 @@ modules = [
 [[package]]
 org = "ballerina"
 name = "task"
-version = "2.3.3"
+version = "2.3.2"
```

Comment:
This is reverted in another commit but it's not getting reflected due to a github downtime. Ignore for now.

Reply:
Ok, seems these versions are not available in Central. I'm committing what I get by building using Central.